src/main/java/com/currencycloud/client/Utils.java

```diff
@@ -1,39 +1,33 @@
 package com.currencycloud.client;
 
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 public enum Utils {
     ;
 
     public static final String dateFormat = "yyyy-MM-dd'T'HH:mm:ssX";
 
-    public static String join(Iterable<String> strings, String separator) {
-        if (strings == null) {
+    public static String join(List<String> strings, String separator) {
```

Comment:
You are changing the signature of a public method that may be used by clients.

Reply:
Good point. Makes more sense to make the other method use Iterable (which would be backwards compatible)

Reply:
You can mark the old method deprecated if it is not being used internally